docs/experience/resources.md

```diff
@@ -0,0 +1,70 @@
+title: Resource Timing
+description: How Sematext Experience collects external resource data such as downloaded images, CSS files, and JavaScript files
+-----
+
+Experience collects resource timing data such as downloaded images, CSS and JavaScript files. When someone visits your website the browser will begin downloading resources. These resources can have a big impact on your website's loading speed.
+
+The same process happens in single-page apps when user navigates to a different part of the web app.
```

Comment:
The same process happens in single-page apps when user navigates to a different part of the web app.
=>
The same process happens in single-page apps when users navigate to a different part of the web app.